homeassistant/components/modbus/bit_switch.py

```diff
@@ -0,0 +1,201 @@
+"""Support for Modbus switches."""
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from datetime import timedelta
+import logging
+from typing import Any
+
+from pymodbus.exceptions import ConnectionException, ModbusException
```

Comment:
You only need ModbusException (which is also on its way out, pending merge of my PR).